packages/react-resize-handle/src/hooks/useResizeHandle.ts

```diff
@@ -280,12 +292,15 @@ export const useResizeHandle = (params: UseResizeHandleParams) => {
           detachMouseHandlers(node);
           detachKeyboardHandlers(node);
         }
+
         attachMouseHandlers(node);
         attachKeyboardHandlers(node);
+
         updateElementsAttrs({
           type: EVENTS.handleRef,
           event: new CustomEvent(EVENTS.handleRef),
-          value: currentValue.current,
+          value: UNMEASURED,
```

Comment:
There and later, we cannot use stale (old) values if a `ref` was remounted: it's either a new element or options were updated